opal/datatype/opal_copy_functions_heterogeneous.c

```diff
@@ -141,106 +592,183 @@ static inline void opal_dt_swap_long_double(void *to_p, const void *from_p, cons
  */
 #define COPY_TYPE_HETEROGENEOUS(TYPENAME, TYPE) COPY_TYPE_HETEROGENEOUS_INTERNAL(TYPENAME, TYPE, 0)
 
+#if 0
+Summaryizing the logic of the pFunc copy functions
+with regard to long doubles:
+
+For terminology I''ll use
```

Comment:
typo (double single quote)

Reply:
Some versions of gcc complain about unmatched quotes inside "#if 0" blocks, but I guess the solution is I just shouldn't be using "#if 0" to make my comment blocks.  I'll switch